lib/cmd/fh3/build.js

```diff
@@ -237,19 +256,13 @@ function doBuild(params, cb) {
       if (keys.length > 0) {
         async.map(keys, common.waitFor, function(err, results) {
           if (err) {
+            if ( err[0].error === "Error status returned from SCM: 500") {
+              return cb(i18n._("Not found the git repository. Check the git parameters informed."));
```

Comment:
Wdyt about this message
`Git repository not found. Please check that all provided Git parameters are valid.`

Reply:
👍 Done Tks. Could you approve now?